src/main/java/com/github/wovnio/wovnjava/HtmlConverter.java

```diff
@@ -168,9 +162,9 @@ private void appendHrefLang() {
     }
 
     private void replaceNodeToMarkerComment(Element element) {
-        Comment comment = new Comment(WOVN_MARKER_PREFIX + String.valueOf(markers.size()));
-        element.replaceWith(comment);
-        markers.add(restore(element.outerHtml())); // restore original text if element has marker
+        String commentKey = htmlReplaceMarker.addCommentValue(htmlReplaceMarker.revert(element.html()));
```

Comment:
Is `htmlReplaceMarker.revert` needed?

Reply:
Yes, otherwise depending on the DOM traversal order, an `Element` might already have its child element commented out. We don't want that to happen.

Reply:
I got it.
So, wovnjava is doing useless marking and useless reverting 😢 
We can leave it now.

Reply:
Yeah. Sometimes you see only 3 comments on the final HTML but they are number 7, number 9, number 11 etc. It is because of that.